sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryPipelineResponse.java

```diff
@@ -20,4 +32,29 @@ public int getStatusCode() {
     public String getBody() {
         return body;
     }
+
+    public Set<String> getErrors() {
+        return parseErrors(body); // parseErrors on demand
+    }
+
+    public boolean invalidInstrumentationKey() {
+        Set<String> errors = parseErrors(body);
+        return errors != null && errors.contains(INVALID_INSTRUMENTATION_KEY);
```

Comment:
this is checking if the Set of errors _exactly_ contains the message

maybe you want to check if any of the errors contains the text message?

Reply:
errors is a set. it will return true if it contains that message, not contains exactly

Reply:
i'm not sure I explained my question above

were you able to test this, does breeze return body exactly equal to "Invalid instrumentation key"?

Reply:
the test body i used in end-to-end test was from breeze. it contains 1 message in errors, even if it returns multiple messages, getErrors still works.